### ISSUE TITLE
grml-x crashes on start when running under qemu

### ISSUE THREAD

**Issue by br-olf** [grml/grml#152]
I  encountered a crash in grml-x using grml64 in qemu-kvm on arch linux. The crash seems to be related to an incomplete python2 to python3 upgrade.

I used the following command to boot into grml:
```
qemu-system-x86_64 -enable-kvm -smp 1 -m 2048 -boot d -cdrom ~/Downloads/grml64-full_2020.06.iso  -drive if=virtio,file=disk.raw  -net nic,model=virtio -net user
```
The error looks like:
![grml-x error](https://user-images.githubusercontent.com/7143440/86767797-d1e95a00-c04c-11ea-8b53-fa82304dba97.png)

**Comment by mika** [comment]
Thanks for providing the full command line, this helped me reproduce the issue.

**Comment by mika** [comment]
This is fixed with the upload of grml-x 0.7.2 (reachable via `apt update; apt install grml-x` on Grml live system).
Thanks for reporting this, I also added it to the list of known Grml 2020.06 bugs at https://grml.org/bugs/known/

**Comment by br-olf** [comment]
Thanks for the fast fix :+1:
